packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react.test.ts

```diff
@@ -371,6 +381,26 @@ describe('amplify render tests', () => {
       });
     });
 
+    describe('GraphQL', () => {
+      it('DataStoreCreateItem', () => {
+        expect(
+          generateWithAmplifyRenderer('workflow/dataStoreCreateItem', rendererConfigWithGraphQL),
+        ).toMatchSnapshot();
+      });
+
+      it('DataStoreUpdateItem', () => {
+        expect(
+          generateWithAmplifyRenderer('workflow/dataStoreUpdateItem', rendererConfigWithGraphQL),
+        ).toMatchSnapshot();
+      });
+
+      it('DataStoreDeleteItem', () => {
```

Comment:
Can we give better names to these tests? Are we just testing that we can codegen without an error?

Reply:
These mirror the block of DataStore tests directly above them. The intent of these tests appear to be to verify that there is no error when rendering and to lock-in the snapshot as the expected output